odbc/src/api/data.rs

```diff
@@ -108,24 +108,7 @@ impl IntoCData for Bson {
 
     fn to_binary(self) -> Result<Vec<u8>> {
```

Comment:
Should we just move everything into json and return the json bytes? This would ensure any transformation on it to text is successful.

Reply:
Made the change 👍